chad/chadHomeworkWk2

```diff
@@ -0,0 +1,176 @@
+//==================== ** 1 ** ====================//
+
+**a)**
+
+  function fractionString(n, d) {
+    var rem = n%d;
+    var whole = Math.floor(n/d);
+    var proper = whole + ' ' + rem + '/' + d;
+    return proper;
+  }
+  console.log(fractionString(3, 3));
+
+**b)**
+
+  function fractionString(n, d) {
+    var rem = n%d;
+    var whole = Math.floor(n/d);
+    if (whole === 0) {
+      var proper = n + '/' + d;
+    } else if (rem === 0) {
+        var proper = whole;
+      } else {
+          var proper = whole + ' ' + rem + '/' + d;
+        }
+    return proper;
+  }
+  console.log(fractionString(3, 3));
+
+
+//==================== ** 2 ** ====================//
+
+**a)**
+
+  var i = (n - Math.floor(n)) >= .5 ? Math.ceil(n) : Math.floor(n); 
+
```

Comment:
Nice work on this one, but to simplify even further you can actually just write:  var i = Math.round(n);